src/script/components/panel/ConversationProtocolDetails/ConversationProtocolDetails.tsx

```diff
@@ -81,14 +81,14 @@ const wrapperStyles: CSSObject = {
 
 const ConversationProtocolDetails: React.FC<ConversationProtocolDetailsProps> = ({protocol, cipherSuite}) => {
   return (
-    <>
+    <div>
       <div className="conversation-details__list-head">{t('conversationDetailsProtocolDetails')}</div>
       <div css={wrapperStyles}>
         <div css={titleStyles}>Protocol</div>
         <div css={subTitleStyles} data-uie-name="protocol-name">
-          {protocol}
+          {protocol.toUpperCase()}
         </div>
-        {protocol.toLocaleLowerCase() === ConversationProtocol.MLS && cipherSuite && (
+        {protocol === ConversationProtocol.MLS && cipherSuite && (
```

Comment:
We were passing already uppercased version before. Knockout-rendered components do not check prop type validity. We have to pass `ConversationProtocol` now (which is lowercase) and render it uppercased.